src/python/WMCore/WMSpec/WMTask.py

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env python
+# pylint: disable=W0212
+# W0212: Names are not accessible for WMTask objects, etc.
```

Comment:
I'm usually against disabling pylints because it might actually be needed somewhere in the future, but this seems to be a good candidate. Maybe you could just add the pylint type to this line too (protected-access)